### DISCUSSION TITLE
What is the formal order of accuracy of FV Navier Stokes kernels?

### DISCUSSION THREAD

**Discussion by aiskhak** [idaholab/moose#20031] [Q&A General] [answered]
Hello all,

I need to perform Richardson extrapolation, which requires knowledge of formal order of accuracy (in space and in time). However, I did not find it in the documentation.

Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

In space:
If you use Rhie-Chow interpolation for velocity and 'average' for the advected quantity you can get second order accuracy.
But our most stable configuration right now is with first order upwing for the advected quantity, which will only give you first order accuracy. We are planning to add more second order schemes with limiters to avoid oscillations soon. 

In time:
it ll depend on your time integration scheme. If you have not specified anything, you are using first order implicit Euler. There are second order schemes available as well, see: https://mooseframework.inl.gov/syntax/Executioner/TimeIntegrator/index.html

Guillaume
